examples/pxScene2d/external/node-v8.15.1_mods.patch

```diff
@@ -1,324 +1,5 @@
-diff --git a/examples/pxScene2d/external/libnode-v8.15.1/common.gypi b/examples/pxScene2d/external/libnode-v8.15.1/common.gypi
```

Comment:
Is there any reason why you are reverting so many changes from node-v8.15.1_mods.patch?

Reply:
I will re-check and re-raise. No code in current patch need to to be removed.